apps/wing/src/util.ts

```diff
@@ -95,4 +96,22 @@ export function parseNumericString(text?: string) {
   return number;
 }
 
-export const currentPackage: { name: string, version: string, engines: { node: string } } = require("../package.json");
+/**
+ * Converts a binary string to a base64/base64url string
+ */
+export function base64Encode(stringToEncode: string, url?: boolean): string {
+  return url ? base64url(stringToEncode) : btoa(stringToEncode);
+}
+
+/**
+ * Converts a base64/base64url string to a binary string
+ */
+export function base64Decode(stringToDecode: string, url?: boolean): string {
+  return url ? base64url.decode(stringToDecode) : atob(stringToDecode);
+}
```

Comment:
I don't think a library is needed. I didn't test this, but this is built in since node18 afaik

```suggestion
export function base64Encode(stringToEncode: string, url?: boolean): string {
	return Buffer.from(stringToEncode).toString(url ? "base64url" : "base64");
}

/**
 * Converts a base64/base64url string to a binary string
 */
export function base64Decode(stringToDecode: string, url?: boolean): string {
	return Buffer.from(stringToDecode, url ? "base64url" : "base64").toString("utf8")
}
```